app/controllers/ThresholdSummaryController.scala

```diff
@@ -58,34 +59,52 @@ class ThresholdSummaryController @Inject()(implicit val messagesApi: MessagesApi
     implicit user =>
       implicit request => {
         withCurrentProfile { implicit profile =>
-          getVatThresholdPostIncorp().flatMap {
-            case VatThresholdPostIncorp(true, _) =>
+          getVatThresholdAndExpectedThreshold().flatMap {
+            case Voluntary(a) =>
```

Comment:
It would be more readable with an If-Else in this case instead of using the object Voluntary